staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -139,15 +143,15 @@ func (f *TestFieldManager) Reset() {
 }
 
 func (f *TestFieldManager) Apply(obj runtime.Object, manager string, force bool) error {
-	out, err := fieldmanager.NewFieldManager(f.fieldManager).Apply(f.liveObj, obj, manager, force)
+	out, err := fieldmanager.NewFieldManager(f.fieldManager, nil).Apply(f.liveObj, obj, manager, "", force)
 	if err == nil {
 		f.liveObj = out
 	}
 	return err
 }
 
 func (f *TestFieldManager) Update(obj runtime.Object, manager string) error {
-	out, err := fieldmanager.NewFieldManager(f.fieldManager).Update(f.liveObj, obj, manager)
+	out, err := fieldmanager.NewFieldManager(f.fieldManager, nil).Update(f.liveObj, obj, manager)
```

Comment:
I wish the test `TestFieldManager` could be used to test the feature so that we don't have to strictly rely on the integration test. Can we do this?

Reply:
Added some unit tests